### DISCUSSION TITLE
AUR Package

### DISCUSSION THREAD

**Discussion by 0xGingi** [Radiicall/jellyfin-rpc#49] [General]
I've started maintaining an AUR Package for jellyfin-rpc if anyone is interested

https://aur.archlinux.org/packages/jellyfin-rpc-git

**Comment by justradical** [comment]
I could maybe add this to the readme with credit to you ofc

**Comment by justradical** [comment]
I think the service file can be placed in /etc/systemd/user instead of symlinking it to the user's folder. Would be better for a system-wide install like this

**Reply by 0xGingi** [reply]
I'll change that, thanks

**Reply by 0xGingi** [reply]
Got it changed and uploaded to AUR
